TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/ShooterSystem.java

```diff
@@ -5,18 +5,51 @@
 import com.arcrobotics.ftclib.hardware.motors.MotorEx;
 import com.qualcomm.robotcore.hardware.HardwareMap;
 
+import org.firstinspires.ftc.teamcode.TeleOpMain;
+
+import java.util.function.DoubleSupplier;
+
 public class ShooterSystem extends SubsystemBase {
-    private Motor kuunav;
+    private MotorEx Motor_S;
```

Comment:
Pascal Case? Make this `motorS` or `motor_S`. Even better would be `m_motor`, which is Hungarian notation for a motor that is a member of the class.